integration-tests/deployment/ccip/state.go

```diff
@@ -38,6 +40,14 @@ type CCIPChainState struct {
 	RMNRemote          *rmn_remote.RMNRemote
 	// TODO: May need to support older link too
 	LinkToken *burn_mint_erc677.BurnMintERC677
+	// Map between token Descriptor (e.g. LinkSymbol, WethSymbol)
+	// and the respective token contract
+	// This is more of an illustration of how we'll have tokens, and it might need some work later to work properly.
+	BurnMintTokens677 map[TokenSymbol]*burn_mint_erc677.BurnMintERC677
+	// Map between token Descriptor (e.g. LinkSymbol, WethSymbol)
+	// and the respective aggregator contract
+	Feeds map[TokenSymbol]*aggregator_v3_interface.AggregatorV3Interface
```

Comment:
ah these are all implicitly TokenSymbol to `USD` feeds? Maybe call them USDFeeds in that case